### PR TITLE
Change UI to be more similar to current extension: Account Page subnav 

### PR DESCRIPTION
**Pull request by buberdds**
PR updates styles of Transaction List subnav. Once merge I will have to sync translations files via Transifex. In next iteration this menu will be moved from Account to Stake page.

Screenshot 2022-07-19 at 18 33 50
Screenshot 2022-07-19 at 18 33 43

Screenshot 2022-07-19 at 18 33 17
Screenshot 2022-07-19 at 18 31 55

### CODE REVIEW

**Review by tjanez — APPROVED** [review]
Looks good. And yes, we should move that under the staking tab/subpage in the next iteration.

File: src/app/pages/AccountPage/index.tsx [hunk 1]
```diff
@@ -35,32 +37,45 @@ import { ValidatorList } from '../StakingPage/Features/ValidatorList'
 import { AccountDetails } from './Features/AccountDetails'
 import { AccountSummary } from './Features/AccountSummary'
 
+const StyledNavItem = styled(NavLink)`
+  display: flex;
+  align-items: center;
```
Comment (lukaw3d) [inline src/app/pages/AccountPage/index.tsx:42]: Leftover?

File: src/app/pages/AccountPage/index.tsx [hunk 2]
```diff
@@ -35,32 +37,45 @@ import { ValidatorList } from '../StakingPage/Features/ValidatorList'
...
-        pad="xsmall"
-        // onClick={() => {}} hoverIndicator={{ color: 'background-oasis-blue' }}
-      >
+    <StyledNavItem exact to={route}>
+      <Text>{label}</Text>
+      {!!counter && (
         <Box
-          border={{ side: 'bottom', color: isActive ? 'oasisBlue2' : 'background-contrast-2', size: '2px' }}
-          pad={{ bottom: 'xxsmall', horizontal: 'xxsmall' }}
+          style={{ position: 'absolute', top: '-3px', right: '-5px', zIndex: 2 }}
```
Comment (lukaw3d) [inline src/app/pages/AccountPage/index.tsx:67]: `zIndex: 2` displays above mobile navbar and footer, and doesn't even seem to be needed

File: src/app/pages/AccountPage/index.tsx [hunk 3]
```diff
@@ -151,22 +166,37 @@ export function AccountPage(props: Props) {
...
+            wrap
+          >
             <NavItem
-              route={`/account/${address}`}
               label={t('account.subnavigation.transactions', 'Transactions')}
+              route={`/account/${address}`}
             />
+
             <NavItem
+              counter={stake.delegations?.length || 0}
```
Comment (lukaw3d) [inline src/app/pages/AccountPage/index.tsx:183]: I'd remove `|| 0` to still allow differentiating missing data

**Review by lukaw3d — APPROVED** [review]

File: src/app/pages/AccountPage/index.tsx (hunk 2 above)
Comment (buberdds) [reply src/app/pages/AccountPage/index.tsx]: damn it it is needed. will include a fix in the last "change ui" PR

Comment (lukaw3d) [reply src/app/pages/AccountPage/index.tsx]: hm, when is it needed?
z

Comment (buberdds) [reply src/app/pages/AccountPage/index.tsx]: When a link has a counter and next link in DOM hierarchy is active.
In our case: when debonding page is active and Active delegation has a counter.

**Pull request merged**